tests/dummy/app/styles/app.scss

```diff
@@ -26,3 +26,70 @@
 .custom-row {
   background: green;
 }
+
```

Comment:
This should all be moved to `/app/styles/ember-table/default.scss`, not in the Dummy app. Check out how `ice-pop` and other internal addons work, this allows users to import the styles optionally.